client/src/plugins/version-info/ReleaseInfo.less

```diff
@@ -1,9 +1,10 @@
 :local(.ReleaseInfo) {
   overflow: auto;
+  max-height: 500px;
 
   h1, h2, h3, h4 {
     font-size: inherit;
-    font-weight: bold;
+    font-weight: 500;
```

Comment:
Is that the default font weight? Let us not deal with magic numbers here but instead (re)set the font weight in a single place.